Support/Python/unitybuild/vcs.py

```diff
@@ -30,36 +30,33 @@ def git(cmd, cwd=None):
 Raises CalledProcessError if process cannot be started, or exits with an error."""
   if cwd is None:
     cwd = os.getcwd()
-  if type(cmd) is str:
+  if isinstance(cmd, str):
     cmd = ['git'] + cmd.split()
   else:
     cmd = ['git'] + list(cmd)
 
   try:
-    proc = Popen(cmd, cwd=cwd, stdout=PIPE, stderr=PIPE)
+    proc = subprocess.Popen(cmd, cwd=cwd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding='utf-8')
```

Comment:
@b-camacho , @billyquith, I changed how this is done (note the lack remove of `str(stdout)` below. Can you confirm that this still works for you (especially on Windows)?

Without this change, none of the regexes were matching, since the strings were `b"foo bar"` instead of just a proper python 3 `"foo bar"`